### PR TITLE
Apac community

### PR DESCRIPTION
**Pull request by furecool**
## Types of changes

* **New feature**

## Description

Add APAC community page

## Steps to Test This Pull Request

1. Open about/apac-community
2. Click pycon community cards

## Related Issue

#296

### CODE REVIEW

File: pages/about/apac-community.vue [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                            class="popup"
+                            :class="{
+                                'popup--show': popupIndex === index,
+                            }"
+                            @click="popupClose(index)"
+                        >
+                            <div class="popup__content">
+                                <button
+                                    ref="popupBtn"
+                                    class="popup__content__button"
```
Comment (mattwang44) [inline pages/about/apac-community.vue:45]:
```suggestion
                                    class="popup__contentButton"
```

File: pages/about/apac-community.vue [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                                        w-20
+                                        lg:w-32
+                                        h-20
+                                        lg:h-32
+                                    "
+                                />
+                                <h4
+                                    class="
+                                        font-bold font-serif
+                                        text:md
```
Comment (mattwang44) [inline pages/about/apac-community.vue:66]: 這個是否怪怪的？是 `md:text-base` 之類的嗎？

Comment (furecool) [reply pages/about/apac-community.vue:66]: 對哦，沒注意到。應該是 `text-base` 沒錯。
不過預設大小就是 `text-base` ，我就直接把他刪掉囉？

Comment (mattwang44) [reply pages/about/apac-community.vue:66]: ok~

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**